### PR TITLE
Prometheus

### PR DESCRIPTION
**Pull request by emosbaugh**

### CODE REVIEW

**Comment by emosbaugh** [comment]
questions:

- do we expose this stuff via port-forward? ingresss? node ports? https://github.com/coreos/kube-prometheus/blob/master/docs/exposing-prometheus-alertmanager-grafana-ingress.md

- grafanas uses admin:admin for creds by default. is this ok?

File: addons/prometheus/0.33.0/Manifest [hunk 1]
```diff
@@ -0,0 +1,8 @@
+grafana grafana/grafana:6.2.2
+alertmanager quay.io/prometheus/alertmanager
+kube-rbac-proxy quay.io/coreos/kube-rbac-proxy:v0.4.1
+kube-state-metrics quay.io/coreos/kube-state-metrics:v1.7.2
+node-exporter quay.io/prometheus/node-exporter:v0.18.1
+k8s-prometheus-adapter-amd64 quay.io/coreos/k8s-prometheus-adapter-amd64:v0.4.1
+prometheus quay.io/prometheus/prometheus
+prometheus-operator quay.io/coreos/prometheus-operator:v0.33.0
```
Comment (areed) [inline addons/prometheus/0.33.0/Manifest]: Doesn't this need the quay.io/coreos/jsonnet-ci image?

Comment (areed) [reply addons/prometheus/0.33.0/Manifest]: I see it's not used at runtime

File: addons/prometheus/0.33.0/install.sh [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    printf "The UIs of Prometheus, Grafana and Alertmanager have been exposed on NodePorts ${GREEN}30900${NC}, ${GREEN}30902${NC} and ${GREEN}30903${NC} respectively.\n"
+    if [ -n "$GRAFANA_ADMIN_PASS" ]; then
+        printf "\n"
+        printf "To access Grafana use the generated user:password of ${GREEN}${GRAFANA_ADMIN_USER:-admin}:${GRAFANA_ADMIN_PASS} .${NC}\n"
+    fi
+    printf "\n"
+    printf "\n"
+}
+
+function prometheus_crd_ready_spinner() {
```
Comment (areed) [inline addons/prometheus/0.33.0/install.sh]: You could also do `spinner_until -1 prometheus_crd_ready`

Comment (emosbaugh) [reply addons/prometheus/0.33.0/install.sh]: do you mean `spinner_until 1 prometheus_crd_ready`?

**Pull request merged**